config/fcos/v1_5_exp/translate.go

```diff
@@ -362,3 +362,28 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func arePartitionEqual(p1, p2 types.Partition) bool {
+    return p1.Label == p2.Label && p1.Number == p2.Number
+}
+
+
+func checkPartitionTable(disk string, wipeTable bool, partitions []types.Partition) error {
+   // if wipetable is true do nothing
+	if wipeTable {
+        return nil
```

Comment:
The indentation here is failing CI tests.  You can run them locally with `./test`.  `gofmt -w <file>` will fix the formatting.